rosdep/osx-homebrew.yaml

```
@@ -691,6 +691,10 @@ python-vtk:
   osx:
     homebrew:
       depends: [libvtk]
+python3:
+  osx:
+    homebrew:
+      packages: []
```

Is the package listed intended to be empty?

---

No, but I went through and recertified different versions to make it load correctly on python3.

---

Looking at https://docs.brew.sh/Homebrew-and-Python#python-3y and https://formulae.brew.sh/formula/python@3.11

It seems like the `python@3` or `python3` alias for the latest python3.x formula is an appropriate definition here.

```suggestion
      packages: [python3]
```